blob.py

```diff
@@ -132,9 +143,11 @@ def create(context, backend, input, output, format, theme, tempo, language):
         music21.converter.parse(input),
         theme=Theme[theme],
         language=language,
-        tempo=tempo
+        tempo=tempo,
+        tracks=tracks,
+        phoneme_fill_in=Phoneme[phoneme_fill_in]
```

Comment:
See above; I usually prefer shorter names whenever possible, but that isn't written in stone. Feel free to keep it as-is if you have strong opinions about that.

```suggestion
        tempo=tempo,
        tracks=tracks,
        fill=Phoneme[fill]
```